wasm/pre-module.js

```diff
@@ -15,6 +31,10 @@ var initStdInOutErr = function() {
   var input = "Hola mundo\n";
```

Comment:
Do we still need `var input = "Hola mundo\n";` here?

Reply:
Nope